controllers/cloudCodeController.js

```diff
@@ -0,0 +1,19 @@
+const axios = require('axios');
+const HttpError = require('../models/httpError');
+const { createURL } = require('../utils/helper');
+
+const getCloudCodeFunctions = (req, res, next) => {
```

Comment:
Maybe name this runCloudCodeFunctions? I thought this was going to be a GET request but we'll have to send params so it'll be a POST. That's my mistake.